packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRow.tsx

```diff
@@ -162,8 +170,11 @@ export function TransactionsTableRow({
       data-testid={`${isClaimableTx ? 'claimable' : 'deposit'}-row-${tx.txId}`}
       className={twMerge(
         'relative mx-4 grid h-[60px] grid-cols-[140px_140px_140px_140px_100px_180px_140px] items-center justify-between border-b border-white/30 text-xs text-white',
-        className
+        className,
+        shouldHighlightTx && 'blink'
       )}
+      onClick={() => setShouldHighlightTx(false)}
+      onMouseOver={() => setShouldHighlightTx(false)}
```

Comment:
i find it strange to go away on click / mouse over
can we not do this and stop blinking automatically as time passes?

Reply:
removed these